BasicSample/Assets/GrabAndThrow/Scripts/CustomCenterOfMass.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.OpenXR.BasicSample
+{
+	[RequireComponent(typeof(Rigidbody))]
+	/// <summary>
+	/// Defines a custom center of mass for a given GameObject's Rigidbody
+	/// </summary>
+	public class CustomCenterOfMass : MonoBehaviour
```

Comment:
why don't I see this getting used? Is this set in the scene automatically for each rigid body?

Reply:
I just double checked, and I guess we don't need this. Originally, I was having issues with the center of mass of these combined objects not getting calculated properly, but it seems to be working fine now. I'll remove this script.